Connected/Assets/Scripts/GarbageBin.cs

```diff
@@ -4,8 +4,8 @@
 
 public class GarbageBin : MonoBehaviour
 {
-    private void OnCollisionEnter(Collision collision)
+    private void OnTriggerEnter(Collider collider)
```

Comment:
Have you tried just this code, with the container's mesh collider disabled?

Reply:
The container’s mesh collider is so that you cannot throw objects straight through the container. 

I want people to throw objects into the container, despite mesh colliders being slow. ^^ If you have an option I’d take it. Convex collider won’t work as the objects need to be able to go inside the container. I want them to be destroyed when they hit the bottom, because it simply is funnier to throw objects into the container than on the container.